client/state/document-head/selectors.js

```diff
@@ -19,7 +21,7 @@ const UNREAD_COUNT_CAP = 40;
  * @param  {Object}  state  Global state tree
  * @return {?String}        Document title
  */
-export function getTitle( state ) {
+export function getDocumentHeadTitle( state ) {
```

Comment:
`client/reader/full-post/controller.js` still imports as `getTitle` in this branch.